### ISSUE TITLE
update?

### ISSUE THREAD

**Issue by larkum** [bepitulaz/Wordpress-Reader-iOS#1]
Any chance this app could be updated to iOS 7 and to include AFNetwork 2?

**Comment by bepitulaz** [comment]
Yes, I consider to update it, but not soon. I'll try to update it next month :)

**Comment by larkum** [comment]
:+1: :smile: Thanks I look forward to it, its a great app, and has solved a massive issue for me thanks again for making this :smile:

**Comment by larkum** [comment]
Any news for an update?
